CHANGELOG.md

```diff
@@ -1,24 +1,49 @@
-# Change Log
-All notable changes to this project will be documented in this file.
+# Changelog
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/)
-and this project adheres to [Semantic Versioning](http://semver.org/).
+<!-- Instructions
+
+This changelog follows the patterns described here: <https://keepachangelog.com/en/1.0.0/>.
+
+Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
+
+-->
+
+The latest published SimpleCSS release is [0.2.1](#021-2021-07-20) which was released on 2021-07-20.
+You can find its changes [documented below](#021-2021-07-20).
 
 ## [Unreleased]
 
 This release has an [MSRV][] of 1.65.
 
-## [0.2.1] - 2021-07-20
-- Add rules sorting by specificity. Thanks to [@baskerville](https://github.com/baskerville)
+This is the first release under the stewardship of [Linebender][], who is now responsible for maintenance
+of this crate. Many thanks to Yevhenii Reizner for the years of hard work that he has poured into this and
+other crates.
+
+## Added
+
+- Support for `no_std`. ([#17][] by [@waywardmonkeys][])
+
+## [0.2.1][] (2021-07-20)
+
+- Add rules sorting by specificity. ([#7][] by [@baskerville][])
+
+## [0.2.0][] (2019-08-17)
 
-## [0.2.0] - 2019-08-17
 - A complete rewrite.
 
-## 0.1.0 - 2017-01-14
+## 0.1.0 (2017-01-14)
```

Comment:
The dash (as opposed to parentheses) separation is what the template uses. Anyway, can discuss that at a later date as it affects more than just this repo.

Reply:
At some point, one of our repos diverged. I'm happy to come to a decision and then just do this however in this and simplecss rather than change it one way and then back. :)

Reply:
Err, this and svgtypes.